src/pages/login.js

```diff
@@ -0,0 +1,76 @@
+import React, {Component} from 'react';
+import {Button, FormField, Text, TextInput} from "grommet";
+import request from "superagent/lib/client";
+
+const api_endpoint = '';
+
+export default class Login extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      error: ''
+    };
+  }
+
+  onChangeEmail = event => {
+    this.setState({email: event.target.value});
+  };
+
+  onChangePassword = event => {
+    this.setState({password: event.target.value});
+  };
+
+  handle_login_response(res) {
+    if (res.hasOwnProperty('jwt')) {
+      this.setState({error: ''});
+      localStorage.jwt = res.jwt;
+      window.location.pathname = '/';
+    } else {
+      this.setState({error: res.error})
+    }
+  }
+
+  getErrorText() {
+    return this.state.error !== '' ? "Invalid email or password" : "";
+  }
+
+  login = () => {
+    /* TODO fix CORS and test this */
```

Comment:
Remove this and do test it please.

Reply:
As I said in the PR message, I can't fully test this because I'm having problems with `CORS`.
`axios` seemed to work for you but @apedrob suggested using `superagent`.
@ZePedroResende @0urobor0s

Reply:
That should be in the PR itself as you put it. Such comments shouldn't go to production.